pkg/detectors/scrutinizerci/scrutinizerci.go

```diff
@@ -0,0 +1,73 @@
+package scrutinizerci
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"scrutinizerci"}) + `\b([0-9a-z]{64})\b`)
```

Comment:
Please change this to be just `scrutinizer` to cache `scrutinizer ci`, scrutinizer-ci`, `scrutinizer`, etc

Reply:
Same with L29 too

Reply:
Commited the changes